.github/ISSUE_TEMPLATE/question.md

```diff
@@ -1,9 +1,9 @@
 ---
 name: Question
-about: The issue tracker is not for questions. Please ask questions on https://stackoverflow.com/questions/tagged/emcb-udp-master or contact Eaton.
+about: The issue tracker is not for questions. Please ask questions on https://stackoverflow.com/questions/tagged/emcb-udp-coordinator or contact Eaton.
 
 ---
 
 🚨 The issue tracker is not for questions 🚨
 
-If you have a question, please ask it on https://stackoverflow.com/questions/tagged/emcb-udp-master or contact Eaton.
+If you have a question, please ask it on https://stackoverflow.com/questions/tagged/emcb-udp-coordinator or contact Eaton.
```

Comment:
Do we want to keep this stackoverflow link? Feels like we should just use the issues section of this repo? Or do we have a better place? The tag has never been used and nobody is monitoring it anyway.

Reply:
@NasibMansour should it be kept or removed?

Reply:
I agree with Michael, let's remove it